### ISSUE TITLE
Ensure all instances of 'master' are renamed to more inclusive synonyms

### ISSUE THREAD

**Issue by xanmanning** [PyratLabs/ansible-role-k3s#62]
In line with the wider tech community, and as a matter of avoiding any unnecessary discomfort or distress, any references to the legacy term "master" should be removed from this project.

K8s has marked this term as legacy, GitHub has renamed the default branch `main` on new repositories with mechanisms for migrating existing repositories arriving by the end of this year.

### Tasks

The following should be evaluated and (hopefully) completed prior to the v2 release of this role.

  - [x] Remove legacy terms from Documentation
  - [x] Remove legacy terms from comments in YAML
  - [x] Remove legacy terms from task names
  - [x] Remove legacy terms from Ansible variables/facts
    - [x] Ensure any variables that are renamed have rename tasks to avoid breakage to existing deployments.
  - [x] Migrate default branch to `main` (Low priority, this should happen automatically)

### Sources

  - https://kubernetes.io/docs/reference/glossary/?fundamental=true
  - https://github.com/github/renaming

**Comment by xanmanning** [comment]
Blocked by issue, migrating to `main` caused import issues in Galaxy - I'm not happy losing stats.

https://github.com/ansible/galaxy/issues/2535

**Comment by xanmanning** [comment]
Done
